frontend/src/metabase/css/core/colors.module.css

```diff
@@ -115,6 +118,11 @@
     var(--mb-color-brand),
     white 45%
   );
+  --mb-base-color-brand-20: color-mix(
```

Comment:
This is used in the icon
https://github.com/metabase/metabase/blob/191d832d8fac25f1c0f510d0ad92b12cc2e460e6/frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.module.css#L3